lib/gems/pending/appliance_console/cli.rb

```diff
@@ -113,6 +113,7 @@ def parse(args)
         opt :username, "Database Username",  :type => :string,  :short => 'U', :default => "root"
         opt :password, "Database Password",  :type => :string,  :short => "p"
         opt :dbname,   "Database Name",      :type => :string,  :short => "d", :default => "vmdb_production"
+        opt :dedicated, "Configure this server as a dedicated database instance", :type => :bool, :short => 'D', :default => false
```

Comment:
This wording was deemed confusing in prior changes. I think we should be consistent and use something like "Run this server as a standalone database server" which comes directly from https://github.com/ManageIQ/manageiq-gems-pending/blob/master/lib/gems/pending/appliance_console/internal_database_configuration.rb#L54